engine/classes/ElggRiverItem.php

```diff
@@ -29,7 +29,7 @@ class ElggRiverItem {
 	public $access_id;
 	public $view;
 	public $posted;
-	public $enabled;
+	protected $enabled;
```

Comment:
I don't think it will reach __get with it defined as protected prop

Reply:
Magic getter should hit: see https://www.php.net/manual/en/language.oop5.overloading.php#object.get

> __set() is run when writing data to inaccessible (**protected** or private) or non-existing properties.

> __get() is utilized for reading data from inaccessible (**protected** or private) or non-existing properties.